lib/smart_app_launch/client_suite/token_use_verification_test.rb

```diff
@@ -8,7 +8,9 @@ class SMARTClientTokenUseVerification < Inferno::Test
     description %(
         Check that a SMART token returned to the client was used for request
         authentication.
-      )
+    )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@93',
```

Comment:
I don't see an explicit test for Bearer needing to be present.
Please confirm and remove @93 if agree.

@93
[When] fetch[ing] FHIR Resources… The [Client] app [SHALL} issue a request that includes an `Authorization` header that presents the `access_token` as a “Bearer” token

Reply:
I wasn't sure because of the removal of the `Bearer` prefix on line 36, but I think I agree that this is not tested. Removed in [9b4d4ad](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/9b4d4adf0a9945b994d01c6ee21808ba05be91a6).